.github/workflows/node.yaml

```diff
@@ -225,13 +237,25 @@ jobs:
         run: |
           echo "Enable build and push: ${{ env.SECRET_ACCESS != '' }}"
           echo "enable=${{ env.SECRET_ACCESS != '' }}" >> $GITHUB_OUTPUT
-      - name: Get the Docker tag
+      - name: Get the Docker tag for GHCR
         id: docker-tag
```

Comment:
should this be called `ghcr-tag` to clarify its purpose?

Reply:
Sure!